msm-sitemap.php

```diff
@@ -627,10 +627,16 @@ public static function build_root_sitemap_xml() {
 	}
 
 	public static function get_sitemap_post_id( $year, $month, $day ) {
+		$ymd = self::get_date_stamp( $year, $month, $day );
+
 		$sitemap_args = array(
-			'year' => $year,
-			'monthnum' => $month,
-			'day' => $day,
+			'date_query' => array(
+				array(
+					'before' => sprintf( '%s 00:00:00', $ymd ),
+					'after' => sprintf( '%s 00:00:00', $ymd ),
```

Comment:
`inclusive` just adds an equality check so we end up with a date query that looks like:

`post_date <= '2016-12-31 00:00:00' AND post_date >= '2016-12-31 00:00:00'`

It's basically a workaround to match an exact date (the `<` and `>` cancel each other out so the `=` is the only thing that's relevant) since WP_Date_Query doesn't have an existing arg for that.